src/main/java/com/example/onlinebookstore/controller/BookController.java

```diff
@@ -29,7 +33,18 @@ public BookDto getBookById(@PathVariable Long id) {
     }
     
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto bookRequestDto) {
+    public BookDto createBook(@RequestBody BookRequestDto bookRequestDto) {
```

Comment:
Please decide naming convention for methods. You have createBook but also you have delete method, why not deleteBook, I suggest to use just delete/create etc, because it is obvious that we work with book entity in BookController